File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Xamarin.md

```diff
@@ -1,30 +1,30 @@
 ---
-title: Convert Word Document to PDF in Xamarin | Syncfusion
-description: Convert Word to PDF in Xamarin using Xamarin Word (DocIO) library without Microsoft Word or interop dependencies.
+title: Convert Word Document to PDF in Xamarin DocIO Library | Syncfusion
+description: Learn all about splitting PDF documents into single pages or multiple pages with Syncfusion’s essential PDF file-format library for Xamarin.
 platform: file-formats
 control: DocIO
 documentation: UG
 ---
 
-# Convert Word Document to PDF in Xamarin
+# Convert Word Document to PDF in Xamarin DocIO Library
 
 Syncfusion DocIO is a [Xamarin Word library](https://www.syncfusion.com/document-processing/word-framework/xamarin/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or **interop** dependencies. Using this library, you can **convert a Word document to PDF in Xamarin**.
 
 ## Steps to convert Word document to PDF in Xamarin
 
 Step 1: Create a new Xamarin.Forms application project.
 
-![Create Xamarin application in Visual Studio](Xamarin_images/Create-Project-WordtoPDF.png)
+![Create Xamarin Word application in Visual Studio.](Xamarin_images/create-project-word-to-pdf.png)
```

Comment:
Please check all keywords. It is not correct

Reply:
resolved